src/lib/es2015.core.d.ts

```diff
@@ -257,13 +267,20 @@ interface NumberConstructor {
 }
 
 interface ObjectConstructor {
+    /**
+     * Copy the values of all of the enumerable own properties from one or more source objects to a
+     * target object. Returns the target object.
+     * @param target The target object to copy to.
+     */
+    assign<T extends {}>(target: T): T;

```

Comment:
There's no "source object" in this overload so the documentation doesn't make sense... Hard to tell what's going on anyway

Reply:
Being the first overload, removing this may confuse users
Maybe we can change it to "zero or more" though

Reply:
What's the point of writing `Object.assign(foo)` without any other parameters? It doesn't copy `foo` anyway.

Reply:
Yes, this overload has no use at all, neither I am having an argument adding this.